sdk/src/main/java/com/entracer/sdk/util/Constants.java

```diff
@@ -9,7 +9,7 @@
      * API urls, and version.
      */
     public static class API {
-        public static final String BASE_PATH = "https://app.entracer.com";
+        public static final String BASE_PATH = "http://crm.orete.org";   //TODO changed https://app.entracer.com
```

Comment:
Lets not change the main constants file! Instead use a local constant in test file for this